charts/nextcloud/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: nextcloud
-version: 5.5.6
-appVersion: 29.0.6
+version: 5.6.0
+appVersion: 30.0.0
```

```suggestion
version: 6.0.0
appVersion: 30.0.0
```
Last time a lot of people complained we didn't make a breaking change, so let's make a breaking change for this major update.

---

Changed as suggested.